truffle/com.oracle.truffle.api.vm/src/com/oracle/truffle/api/vm/SymbolInvokerImpl.java

```diff
@@ -108,15 +114,19 @@ Object convert(VirtualFrame frame, Object obj) {
         private Object convert(VirtualFrame frame, TruffleObject obj) {
             Object isBoxedResult;
             try {
-                isBoxedResult = ForeignAccess.execute(isBoxed, frame, obj);
+                isBoxedResult = ForeignAccess.sendIsBoxed(isBoxed, frame, obj);
             } catch (IllegalArgumentException ex) {
                 isBoxedResult = false;
```

Comment:
Not really, isBoxed should not throw any kind of InteropException at all.
Any binary message should be supported by all TruffleObjects.